rust/examples/sign-request/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use rand;
+use std::env;
+use uuid::Uuid;
+
+const KID: &str = "KID";
+const ACCESS_TOKEN: &str = "ACCESS_TOKEN";
+const PRIVATE_KEY: &str = "PRIVATE_KEY";
+
+const URL: &str = "https://api.truelayer-sandbox.com/test-signature";
+
+#[tokio::main]
+async fn main() {
+    // load env vars
+    let kid = env::var(KID).expect("Missing env var KID");
+    let access_token = env::var(ACCESS_TOKEN).expect("Missing env var ACCESS_TOKEN");
+    let private_key = env::var(PRIVATE_KEY).expect("Missing env var PRIVATE_KEY");
```

Comment:
I think just inline strings would be better here
```suggestion
    let kid = env::var("KID").expect("Missing env var KID");
    let access_token = env::var("ACCESS_TOKEN").expect("Missing env var ACCESS_TOKEN");
    let private_key = env::var("PRIVATE_KEY").expect("Missing env var PRIVATE_KEY");
```